microprofile/tests/tck/tck-cdi/pom.xml

```diff
@@ -28,6 +28,16 @@
     <name>Helidon Microprofile Tests TCK CDI</name>
 
     <dependencies>
+        <dependency>
+            <groupId>io.helidon.microprofile.metrics</groupId>
+            <artifactId>helidon-microprofile-metrics</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
This is part of helidon MP core, no need to add it

Reply:
There are test failures:
```
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.172 s <<< FAILURE! - in org.jboss.cdi.tck.tests.definition.stereotype.inheritance.StereotypeInheritenceTest
[ERROR] org.jboss.cdi.tck.tests.definition.stereotype.inheritance.StereotypeInheritenceTest.arquillianAfterClass  Time elapsed: 0.021 s  <<< FAILURE!
org.jboss.weld.exceptions.UnsatisfiedResolutionException: WELD-001334: Unsatisfied dependencies for type MetricRegistry with qualifiers @RegistryType 
	at org.jboss.weld.bean.builtin.InstanceImpl.checkBeanResolved(InstanceImpl.java:251)
	at org.jboss.weld.bean.builtin.InstanceImpl.get(InstanceImpl.java:113)
```

mvn dependency:tree does not show `helidon-microprofile-metrics`